tests/integration/test_projectconfigs.py

```diff
@@ -128,3 +129,24 @@ def test_invalid_signature(mini_sentry, relay):
 
     assert response.status_code == 401  # Unauthorized
     assert "signature" in response.text
+
+
+def test_broken_projectkey(mini_sentry, relay):
+    relay = relay(mini_sentry, wait_healthcheck=True)
+    mini_sentry.add_basic_project_config(42)
+    public_key = mini_sentry.get_dsn_public_key(42)
+
+    body = json.dumps({"public_keys": ["broken", public_key]})
```

Comment:
should parametrize over all values that we discussed in FMEA

Reply:
thanks. Added everything except:
- encoding, which we ruled out as a non issue
- large strings, as this is handled by store body as an independent concern